### DISCUSSION TITLE
Move model

### DISCUSSION THREAD

**Discussion by alexlopezit** [spatie/laravel-medialibrary#2779] [Q&A]
I would like to know if there is a process to follow when renaming a model namespace due to moving it from `App/Model` to `App/Models/Model`?

The issue is that when I move the model, the `media` table has a reference to the old namespace and therefore all the media items don't return anymore.

**Comment by alexlopezit** [comment]
Does anyone know a work around for this?

**Comment by erikn69** [comment]
Just update the db field with the new namespace, for example using `SUBSTRING_INDEX`
